scripts/apps/search/directives/ItemRepo.js

```diff
@@ -25,6 +25,10 @@ export function ItemRepo(
 
                 searchProviderService.getAllowedProviderTypes().then((providerTypes) => {
                     scope.searchProviderTypes = providerTypes;
+                    scope.providerLabels = {};
+                    scope.searchProviderTypes.forEach((type) => {
```

Comment:
imo this would be better done in `searchProviderService.getAllowedProviderTypes()` so you don't have to duplicate this code. it could populate `type.label` field and just use it.